### PR TITLE
[FIX] edit sample 

### LINKED ISSUE #554: [BUG] Cannot edit samples
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Sample platform commit (found at the bottom of each page) : Latest commit

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/sample-platform/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/sample-platform/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never visited/used the platform.
- [ ] I have used the platform just a couple of times.
- [ ] I have used the platform extensively, but have not contributed previously.
- [x] I am an active contributor to the platform.

---

Picture says more than 1000 words....

Screenshot from 2021-05-31 18-07-55

### PR DESCRIPTION
**Pull request by kvshravan**
**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [x] I have never used the project.
- [x] I have used the project briefly.
- [x] I have used the project extensively, but have not contributed previously.
- [x] I am an active contributor to the project.

---

Fix #554

### CODE REVIEW

**Comment by kvshravan** [comment]
One of the reasons we never came across this error is because there are no tests for edit_sample function.

**Comment by thealphadollar** [comment]
> One of the reasons we never came across this error is because there are no tests for edit_sample function.

Oh! Do you think adding a test in the PR will be helpful?

File: mod_sample/forms.py [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 
-from mod_upload.forms import CommonSampleForm
+from mod_upload.models import Platform
 
 
-class EditSampleForm(CommonSampleForm):
+class EditSampleForm(FlaskForm):
     """Form to edit sample."""
 
+    notes = TextAreaField('Notes', [DataRequired(message='Notes are not filled in')])
```
Comment (canihavesomecoffee) [inline mod_sample/forms.py:13]: This is duplication of the fields that are in CommonSampleForm.

I'd rather see the offending fields in that class moved to the appropriate class (take a look at the history and you'll see when they've been added and for what reason).

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]
This is not a good fix. The CommonSampleForm class is there with a reason.

Wrt tests, it'd be nice to have some indeed.

**Comment by kvshravan** [comment]
> > One of the reasons we never came across this error is because there are no tests for edit_sample function.
> 
> Oh! Do you think adding a test in the PR will be helpful?

Yes I will be adding a test.

**Review by canihavesomecoffee — DISMISSED** [review]
LGTM

File: mod_upload/forms.py [hunk 2]
```diff
@@ -99,4 +96,7 @@ def validate_version(form, field) -> None:
 class FinishQueuedSampleForm(CommonSampleForm):
     """Form to finalize sample queue."""
 
+    report = SelectField('Do you want to report Issue on GitHub?', choices=[('n', 'No'), ('y', 'Yes')])
```
Comment (thealphadollar) [inline mod_upload/forms.py:99]:
```suggestion
    report = SelectField('Do you want to report an issue on GitHub?', choices=[('n', 'No'), ('y', 'Yes')])
```

**Review by thealphadollar — APPROVED** [review]
A small minor edit. See if you want to include it. 

Rest, LGTM.

**Pull request merged**